packages/back-end/src/models/ProjectModel.ts

```diff
@@ -31,15 +31,21 @@ function toInterface(doc: ProjectDocument): ProjectInterface {
   return omit(ret, ["__v", "_id"]);
 }
 
+interface createProjectProps {
+  name: string;
+  description?: string;
+  id?: string;
+}
+
 export async function createProject(
   organization: string,
-  data: Partial<ProjectInterface>
+  data: createProjectProps
 ) {
-  // TODO: sanitize fields
   const doc = await ProjectModel.create({
-    ...data,
-    organization,
-    id: uniqid("prj_"),
+    organization: organization,
+    id: data.id || uniqid("prj_"),
+    name: data.name || "",
+    description: data.description || undefined,
```

Comment:
`|| undefined` is redundant, unless you expect `null` or `0` to be passed in, which it shouldn't if our object validation is preventing us from getting unexpected types.

I do prefer this explicit style to the previous destructuring so we know which fields we are getting 👍